client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -249,13 +265,24 @@ export default {
           polygon: JSON.stringify(this.featuresObject)
         }
       })
+      const comments = { ...this.segment.relationships.comments } || null
```

Comment:
I think (and sonarcloud thinks that too) that {} won't resolve to false and with that it will never be null.

Reply:
You're right, this makes no sense. Changed in [48cc5c7](https://github.com/demos-europe/demosplan-core/pull/3468/commits/48cc5c75c358cf4e45eb582697207087f25e3f5e)